src/Symfony/Component/Console/Output/ConsoleOutput.php

```diff
@@ -118,39 +116,39 @@ protected function hasStderrSupport()
     }
 
     /**
-     * Checks if current executing environment is IBM iSeries (OS400), which
-     * doesn't properly convert character-encodings between ASCII to EBCDIC.
-     *
-     * @return bool
+     * @return resource
      */
-    private function isRunningOS400()
+    protected function openOutputStream()
     {
-        $checks = array(
-            function_exists('php_uname') ? php_uname('s') : '',
-            getenv('OSTYPE'),
-            PHP_OS,
-        );
+        $outputStream = $this->hasStdoutSupport() ? 'php://stdout' : 'php://output';
 
-        return false !== stristr(implode(';', $checks), 'OS400');
+        return @fopen($outputStream, 'w') ?: fopen('php://output', 'w');
     }
 
     /**
      * @return resource
      */
-    private function openOutputStream()
+    protected function openErrorStream()
```

Comment:
In my opinion you shouldn't modify code just for tests. If this is something you'd like to mock or control in tests, you should inject an object that has the open methods.